ivy/functional/frontends/numpy/random/functions.py

```diff
@@ -197,3 +197,11 @@ def logistic(loc=0.0, scale=1.0, size=None):
     u = ivy.random_uniform(low=0.0, high=0.0, shape=size, dtype="float64")
     x = loc + scale * ivy.log(u / (1 - u))
     return x
+
+
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def laplace(loc=0.0, scale=1.0, size=None):
+    u = ivy.random_uniform(low=0.0, high=1.0, shape=size, dtype="float64")
+    x = ivy.exp(-abs(u - loc) / scale)/ (2 * scale)
+    return x
```

Comment:
We need a line of whitespace at the end of the file

Reply:
> 

@Sam-Armstrong Thanks a lot. Due to branch conflicts, the older commits were deleted. I have generated this [PR](https://github.com/unifyai/ivy/pull/21747) with the changes you suggested. Please review It.